src/Pages/About/Safarri/Safarri.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Saf1 from '../../../asserts/images/saf1.png';
+import Saf2 from '../../../asserts/images/saf2.png';
+import Saf3 from '../../../asserts/images/saf3.png';
+import './Safarri.css';
+
+export default function Safarri() {
+  return (
+    <div className="safarri">
+      <h1 className="xaf">Welcome to Africa Wizzy Safari</h1>
+      <hr />
+      <div className="safarri__holder">
+        <div className="safarri__cards">
```

Comment:
This section can be broken down into sub components. To make it more readable. but for now this is acceptable.